datanode/gateway/graphql/resolvers.go

```diff
@@ -698,6 +703,10 @@ func fromPtr[T any](ptr *T) (ret T) {
 	return
 }
 
+func toPtr[T any](val T) *T {
```

Comment:
I think we have this already in `lib/ptr/ptr.go` that you could pull in.

Reply:
Nice. Somehow I couldn't find it